pipeline/backend/kubernetes/utils.go

```diff
@@ -10,8 +12,16 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+var dnsPattern = regexp.MustCompile(`[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*`)
```

Comment:
```suggestion
var (
  dnsPattern = regexp.MustCompile(`[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*`)
  ErrDnsPatternInvalid = errors.New("name not valid kubernetes DNS name")
)
```
:point_up: this + `make format`